hummingbird/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from hummingbird.models import UserProfile, UserDevice
+
+class UserProfileAdmin(admin.ModelAdmin):
+	list_display = ('name','last_played','song','length',)
+
+admin.site.register(UserProfile,UserProfileAdmin)
+admin.site.register(UserDevice)
+# Register your models here.
```

Comment:
nit: Let's remove this comment. It's not really relevant anymore, now that you've inserted something into the file.